soc/arm/arm/mps2/soc.c

```diff
@@ -56,7 +51,7 @@ void wakeup_cpu1(void)
 	/* Set the Initial Secure Reset Vector Register for CPU 1 */
 	*(uint32_t *)(SSE_200_SYSTEM_CTRL_INITSVTOR1) =
 		CONFIG_FLASH_BASE_ADDRESS +
-		BL2_HEADER_SIZE +
+		CONFIG_ROM_START_OFFSET +
```

Comment:
@microbuilder @anangl @oyvindronningstad may I ask you to re-review the PR and in particular, these additional changes in the soc.c for musca and mps2?

AFAIR, @microbuilder added these for BL2 header, which, now is configurable in TFM and is reflected in ROM_START_OFFSET. So the initial reset vector table needs to be offset by 0x400 only if BL2 is built, when building with TFM. Otherwise not.

Reply:
Can this be done with linker symbols instead?

Reply:
for example? _vector_start?

Reply:
Yep

Reply:
done now, thanks @oyvindronningstad